### ISSUE TITLE
KSAnnotation.arguments include default values

### ISSUE THREAD

**Issue by ansman** [google/ksp#1175]
Perhaps this is intentional but it'd like to check if an annotation argument was explicitly provided or not but it seems that `KSAnnotation.arguments` contains one value per property, even if it wasn't provided at the call site.

Here is an example:
```kotlin
annotation class Test(
  val someProperty: Int = 0
)

@Test
class SomeType
```
Reading `Test` of `SomeType` gives me an argument list of `someProperty = 0`.

**Comment by neetopia** [comment]
yes, this is to ensure processors have access to the default value. You can tell if it is default by `KSValueArgument.isDefault()` function.

**Comment by ansman** [comment]
Ah I missed that. Thanks!
